Userland/Libraries/LibCore/SocketLikeIODevice.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021, Ali Mohammad Pur <mpfard@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibCore/IODevice.h>
+
+namespace Core {
+
+class SocketLikeIODevice : public virtual IODevice {
+    C_OBJECT_ABSTRACT(SocketLikeIODevice)
```

Comment:
Let's put an `;` at the end of the line for consistency with the rest of the codebase.